### PR TITLE
Added SO and Text Layer related change

### PR DESCRIPTION
**Pull request by khound**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -148,7 +149,17 @@ The typical workflow involves retrieving a PSD document manifest file via `/docu
 
 For the time being clients can only use assets stored on EITHER Adobe's Creative Cloud OR external storage (like AWS S3 or Azure Blog Storage).  Support for mixing and matching will be added in the future
 
-### Fonts
+### Text Layer
+
+The Photoshop APIs currently support creating and editing of Text Layer with different fonts, character styles and paragraph styles.
+
+Please see the api docs for more information.
+https://git.corp.adobe.com/pages/dice/pie-in-the-sky/#api-Photoshop-document_operations
```
Comment (noahhoffman) [inline README.md]: @khound link is wrong, it points to the private page

File: README.md [hunk 2]
```diff
@@ -148,7 +149,17 @@ The typical workflow involves retrieving a PSD document manifest file via `/docu
 
 For the time being clients can only use assets stored on EITHER Adobe's Creative Cloud OR external storage (like AWS S3 or Azure Blog Storage).  Support for mixing and matching will be added in the future
 
-### Fonts
+### Text Layer
+
+The Photoshop APIs currently support creating and editing of Text Layer with different fonts, character styles and paragraph styles.
```
Comment (noahhoffman) [inline README.md]: @khound "Text Layer" should be "text layers"

File: README.md [hunk 3]
```diff
@@ -167,6 +178,10 @@ The Photoshop APIs currently support creating and editing of Embedded Smart Obje
 - In order to update an embedded smart object that is referenced by multiple layers you only need to update one of those layers, the effect will be reflected in all layers referencing the same smart object.
 
 Please see the api docs for more information.
+https://git.corp.adobe.com/pages/dice/pie-in-the-sky/#api-Photoshop-document_operations
```
Comment (noahhoffman) [inline README.md]: @khound - same private link problem

File: README.md [hunk 4]
```diff
@@ -167,6 +178,10 @@ The Photoshop APIs currently support creating and editing of Embedded Smart Obje
...
+
+Please find a JSON example of replacing a Smart Object within a layer.
```
Comment (noahhoffman) [inline README.md]: @khound I would just make this a live link: 
"[Smart Object Example Code](https://github.com/AdobeDocs/photoshop-api-docs#example-6-swapping-the-image-in-a-smart-object-layer)

File: README.md (hunk 3 above)
Comment (noahhoffman) [reply README.md]: I would also just make this a link:
"[API Documentation](https://adobedocs.github.io/photoshop-api-docs/#api-Photoshop-document_operations)

File: README.md (hunk 1 above)
Comment (khound) [reply README.md]: @noahhoffman Yes. Thanks for pointing out.

File: README.md (hunk 3 above)
Comment (khound) [reply README.md]: Good suggestion.changing both.

**Comment by khound** [comment]
Implemented all review comments.

**Pull request merged**